jerry-debugger/jerry-client.py

```diff
@@ -68,13 +70,15 @@ def parse_source(debugger, data):
     source_name = ''
     function_name = ''
 
-    function_list = []
```

Comment:
JerryDebuger should be an entity, so there is no point to take this out to global level. Global variables are not recommended in general.